### PR TITLE
chore: update bulk insert csv tool to new format and more normalizing

### PR DESCRIPTION
**Pull request by Goncharo**
refs. metriport/metriport-internal#690

### Dependencies

n/a

### Description

update bulk insert csv tool to new format and more normalizing

### Release Plan

asap, nothing special

### CODE REVIEW

File: packages/utils/src/bulk-insert-patients.ts [hunk 1]
```diff
@@ -54,38 +54,77 @@ function sleep(ms: number) {
...
+function toTitleCase(str: string): string {
+  return str
+    .toLowerCase()
+    .split(" ")
+    .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(" ");
+}
+
+function normalizeGender(gender: string): "M" | "F" {
+  const lowerGender = gender.toLowerCase();
```
Comment (leite08) [inline packages/utils/src/bulk-insert-patients.ts:66]: NABD, but it makes it more robust, add `.trim()`

File: packages/utils/src/bulk-insert-patients.ts [hunk 2]
```diff
@@ -54,38 +54,77 @@ function sleep(ms: number) {
...
+}
+
+function normalizeEmail(email: string): string {
+  return email.toLowerCase();
+}
+
+function normalizeState(state: string): USState {
+  if (
+    state.length === 2 &&
+    Object.values(states).includes(USState[state as keyof typeof USState])
```
Comment (leite08) [inline packages/utils/src/bulk-insert-patients.ts:94]: NABD b/c `USState[..]` contains the 2 letters for states, but we could simplify this by just checking `if (Object.values(states).includes(state))`, right?

File: packages/utils/src/bulk-insert-patients.ts [hunk 3]
```diff
@@ -54,38 +54,77 @@ function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+function toTitleCase(str: string): string {
+  return str
+    .toLowerCase()
+    .split(" ")
+    .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(" ");
```
Comment (leite08) [inline packages/utils/src/bulk-insert-patients.ts:62]: also add `.trim()`?

**Review by leite08 — APPROVED** [review]
LGTM, just small suggestions

**Review by RamilGaripov — APPROVED** [review]
LGTM

File: packages/utils/src/bulk-insert-patients.ts (hunk 2 above)
Comment (Goncharo) [reply packages/utils/src/bulk-insert-patients.ts:94]: yes good point

**Pull request merged**